RecoHI/HiMuonAlgos/python/RecoHiMuon_EventContent_cff.py

```diff
@@ -1,54 +1,52 @@
 import FWCore.ParameterSet.Config as cms
 
-#Full Event content 
-RecoHiMuonFEVT = cms.PSet(
-    outputCommands = cms.untracked.vstring()
-    )
-
-#RECO content
-RecoHiMuonRECO = cms.PSet(
-    outputCommands = cms.untracked.vstring()
-    )
-
 #AOD content
 RecoHiMuonAOD = cms.PSet(
     outputCommands = cms.untracked.vstring()
     )
-
 #Add Isolation
 from RecoMuon.MuonIsolationProducers.muIsolation_EventContent_cff import *
```

Comment:
Where are these ```RecoMuonIsolationXXX``` event contents used here, or in the calling configs?
If I am not wrong this ```import``` could even be removed here, if we want to clean up the config.
@mandrenguyen ?

Reply:
Indeed, it looks like nothing from that import is actually used.    If it's not doing anything, might as well remove it.

Reply:
Okay, I will remove it.